plugins/lsp/lsp-status.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "lsp-status";
+  originalName = "lsp-status.nvim";
+  defaultPackage = pkgs.vimPlugins.lsp-status-nvim;
+  maintainers = [ helpers.maintainers.b3nb5n ];
+  settingsOptions =
+    let
+      indicator_opt = helpers.defaultNullOpts.mkStr "" ''
+        string to show as diagnostics.
+        If you don't have Nerd/Awesome Fonts you can replace defaults with ASCII chars.
+      '';
+    in
+    {
+      kind_labels =
+        helpers.defaultNullOpts.mkAttrsOf types.str ""
+          "An optional map from LSP symbol kinds to label symbols. Used to decorate the current function name.";
```

Comment:
I think @GaetanLepage meant to use multiline strings for all option descriptions, not just `indicator_opt`.

Reply:
updated all the description strings

Reply:
Sorry to nit-picked, but we usually prefer to format the description as follows:

```nix
helpers.defaultNullOpts.mkAttrsOf types.str "" ''
  An optional map from LSP symbol kinds to label symbols. Used to decorate the current function name.
'';
```

i.e. with the description text on a separate line to the quotes.